design/secrets_provider_milestone1_design.md

```diff
@@ -42,54 +42,103 @@
 
 ## Background
 
-The Cyberark Secrets Provider for K8s enables customers to use secrets stored and managed in Conjur/DAP vault and consume them as K8s Secrets for their application containers. During Phase 1, an *init* container was deployed in the same Pod as the customer's applications and populate the K8s secrets defined in the customer's manifest. 
+The Cyberark Secrets Provider for K8s enables customers to use secrets stored and managed in Conjur/DAP vault and consume them as K8s Secrets for their application containers.
+During Phase 1, an *init container* was deployed in the same Pod as the customer's applications and populate the K8s secrets defined in the customer's manifest.
 
 ### Motivation
 
-The current implementation makes the deployment of the Secrets Provider does not support secret rotation and makes the deployment of the Secrets Provider deeply coupled to the customer's applications which not all customers want.
+The current implementation of the Secrets Provider only supports one app and makes the deployment of the Secrets Provider deeply coupled to the customer's applications.
+Also, it launches once and does not support secret value rotation while the app is running.
```

Comment:
@abrahamko this is misleading because in our solution here we also aren't going to "support secret value rotation while the app is running"